CarePartner/Views/FolloweeSummaryView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  FolloweeSummaryView.swift
+//  CarePartner
+//
+//  Created by Pete Schwamb on 3/27/23.
+//  Copyright © 2023 Tidepool Project. All rights reserved.
+//
+
+import SwiftUI
+
+struct FolloweeSummaryView: View {
+
+    var account: FollowedAccount
+
+    var body: some View {
+        VStack {
+            HStack {
+                HStack {
+                    Image(systemName: "person.fill")
+                    Text(account.name)
+                        .font(.headline)
+                }
+                Spacer()
+                HStack {
+                    Image(systemName: "arrow.triangle.2.circlepath")
+                    Text("12:34")
+                }
+                .font(.caption)
+                .foregroundColor(.secondary)
+                Spacer()
+                Image(systemName: "chevron.down")
+                    .foregroundColor(.secondary)
+                    .padding(5)
+            }
+            .padding(.top, 8)
+
+            HStack {
+
+                Text("100 mg/dl")
+                    .font(.headline)
+                    .frame(maxWidth: .infinity)
+                    .padding()
+                    .background(.background)
+                    .frame(height: 44)
+                    .cornerRadius(22)
+                LoopCircleView(closeLoop: true, lastLoopCompleted: Date(), dataIsStale: false)                                .padding(20)
+                Text("0.45 U/hr")
```

Comment:
I assume these are just placeholders? maybe include a comment or TODO? Otherwise use the `currentBG`, `basalRate`, and `lastRefresh` of the FollowedAccount.

Reply:
Yeah, these are placeholders. This ticket doesn't include data fetch/display.